includes/classes/admin/modules/class-tailwind.php

```diff
@@ -15,11 +15,31 @@ public function __construct() {
                 return;
             }
 
+            // WP hooks
+            add_filter( 'language_attributes', array( $this, 'add_pip_prefix_class' ), 20, 2 );
```

Comment:
I don't think it's necessary to add this hook here, maybe it can be managed by within the project itself, because adding pip as prefix can then be managed though both the added hooks `pip/tailwind_api/front_build_args` and `pip/tailwind_api/admin_build_args`